Automattic/wp-calypso
# Login social first

With this PR we will implement 4lWBkrFl1jQuN6irMuoLys-fi-286%3A713.

## Before

<img width="800" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/1868f149-9feb-4773-bb22-0960fcf20e3a">

## After

<img width="800" alt="image" src="https://github.com/Automattic/wp-calypso/assets/52076348/563ebdad-293f-4f65-9dbc-0519698661b1">

## Setting up the Experiment

Follow this (_Testing logged-out homepage implementations_): PCYsg-SwK-p2
With this experiment: 21539-explat-experiment

## Testing

1. Use the live link.
2. Visit `/log-in`. Test desktop and mobile.
3. Visit `/log-in/es` and live link version. Test desktop and mobile.
4. Visit `wordpress.com/start/hosting` and live link version. Test desktop and mobile. They should be identical.
5. Visit `wordpress.com/start/hosting/de` and live link version. Test desktop and mobile. They should be identical.
6. Visit `wordpress.com/start` and live link version. Test desktop and mobile. They should be identical.
7. Visit `wordpress.com/start/de` and live link version. Test desktop and mobile. They should be identical.
8. Visit `wordpress.com/start/videopress-account/user` and live link version. Test desktop and mobile.
9. **P2**: [Link](https://wordpress.com/log-in?redirect_to=http%3A%2F%2Fcalypso.localhost%3A3000%2Fstart%2Fp2%2Fp2-confirm-email&from=p2&signup_url=%2Fstart%2Fp2%2Fuser)  and live link version. Test desktop and mobile.
10. **Jetpack**: [Link](http://wordpress.com/log-in/jetpack?site=https%3A%2F%2Ffalse-shrike.jurassic.ninja)  and live link version. Test desktop and mobile.
11. **Crowdsignal**: [Link](http://wordpress.com/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D51d5e1b0d7358dee3de65c45f4b4dc37a3d6e37dab8d6047545f636c2659d104%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1) and live link version. Test desktop and mobile.
12. **Jetpack Cloud**: [Link](http://wordpress.com/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0) 
13. **Woo**: [Signup](https://wordpress.com/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dc92cfd0e05c2bf6e176a5ed1fbf185a99cf22a08cddf2bd631e1362b240d24f1%26redirect_uri%3Dhttps%253A%252F%252Fwoo.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fstart%25252F%252523%25252Finstallation%2526source%253Dsignup_menu%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916) and [Login](https://wordpress.com/log-in?redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dc92cfd0e05c2bf6e176a5ed1fbf185a99cf22a08cddf2bd631e1362b240d24f1%26redirect_uri%3Dhttps%253A%252F%252Fwoo.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fstart%25252F%252523%25252Finstallation%2526source%253Dsignup_menu%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&client_id=50916&signup_url=%2Fstart%2Fwpcc%2Foauth2-user%3Fref%3Doauth2%26oauth2_redirect%3Dhttps%253A%252F%252Fpublic-api.wordpress.com%252Foauth2%252Fauthorize%252F%253Fresponse_type%253Dcode%2526client_id%253D50916%2526state%253Dc92cfd0e05c2bf6e176a5ed1fbf185a99cf22a08cddf2bd631e1362b240d24f1%2526redirect_uri%253Dhttps%25253A%25252F%25252Fwoo.com%25252Fwc-api%25252Fwpcom-signin%25253Fnext%25253D%2525252Fstart%2525252F%25252523%2525252Finstallation%252526source%25253Dsignup_menu%2526blog_id%253D0%2526wpcom_connect%253D1%2526wccom-from%2526calypso_env%253Dproduction%2526jetpack-code%2526jetpack-user-id%253D0%2526action%253Doauth2-login%2526from-calypso%253D1%26oauth2_client_id%3D50916)
14. MailPoet: [link](https://wordpress.com/log-in?client_id=79002&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26redirect_uri%3Dhttps%253A%252F%252Faccount.mailpoet.com%252Flogin%252Fwpcom%252Fcallback%26scope%3Dauth%26client_id%3D79002)
